frontend/src/components/Pages/YTC/Executor/TokenResult.tsx

```diff
@@ -86,15 +85,48 @@ export const TokenResult: React.FC<TokenResultProps> = (props) => {
                             />
                     }
                     {
-                        tokenType === "YToken"
-                            && <YTPriceTag
-                                amount={token.amount}
-                                baseTokenName={props.baseTokenName}
-                                trancheAddress={props.trancheAddress}
-                            />
+                        tokenType === "YToken" && <></>
+                            // && <YTPriceTag
```

Comment:
is this commented out cos of the bug on YT prices?

Reply:
It is. I just added a line comment stating that